cpp/velox/substrait/SubstraitToVeloxPlanValidator.cc

```diff
@@ -476,9 +476,16 @@ bool SubstraitToVeloxPlanValidator::validate(const ::substrait::GenerateRel& gen
     names.emplace_back(SubstraitParser::makeNodeName(inputPlanNodeId, colIdx));
   }
   auto rowType = std::make_shared<RowType>(std::move(names), std::move(types));
+  std::vector<core::TypedExprPtr> expressions;
   if (generateRel.has_generator() && !validateExpression(generateRel.generator(), rowType)) {
-    LOG_VALIDATION_MSG("Input validation fails in GenerateRel.");
-    return false;
+    if (!validateExpression(generateRel.generator(), rowType) {
```

Comment:
Why should we check the `!validateExpression(generateRel.generator(), rowType)` again?

Reply:
Sorry ,cherry pick error.

Reply:
The outer layer judgment should not be verified